frontend/frontend.go

```diff
@@ -59,17 +63,25 @@ func NewFrontend(logger *slog.Logger, listener net.Listener) *Frontend {
 		done:  make(chan struct{}),
 	}
 
+	// Setup metrics middleware
+	metricsMiddleware := MetricsMiddleware{Emitter: emitter}
+
 	mux := NewMiddlewareMux(
 		MiddlewarePanic,
 		MiddlewareLogging,
 		MiddlewareBody,
 		MiddlewareLowercase,
-		MiddlewareSystemData)
+		MiddlewareSystemData,
+		metricsMiddleware.Metrics(),
+	)
 
 	// Unauthenticated routes
 	mux.HandleFunc("/", f.NotFound)
 	mux.HandleFunc(MuxPattern(http.MethodGet, "healthz", "ready"), f.HealthzReady)
 
+	// Expose Prometheus metrics endpoint
+	mux.Handle("/metrics", allowGETOnly(promhttp.Handler()))
```

Comment:
```suggestion
	mux.Handle(MuxPattern(http.MethodGet, "metrics", promhttp.Handler())

Reply:
Sorry I wasn't clear in last review.  I think this leverages patterns we're using above, and no need for the `allowGETOnly` function.